index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/Style.css">
+    <title>QUIZY BUZZY</title>
+</head>
+<body>
+   <header>
+       <section class="nav"><nav id="nav"></nav></section>
+       <section class="second-header"><div id="second-header"></div></section>
```

Comment:
we should not use the same name twice for a class and an id,

Reply:
got it ..  will be changed later